azerothcore/azerothcore-wotlk
# fix(Scripts/Items): Powerful Anti-venom

## Changes Proposed:

what it do.

missing script for the spell u dig. just what followed what was already done for lesser anti-venom shit.

on god

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/7781
- Closes https://github.com/chromiecraft/chromiecraft/issues/1662

## How to Test the Changes:


1. Train First Aid to 300 to be able to use the item (.setskill 129 300)
2. .additem 19440 10
3. .aura 744 / 4940 / 6411 / 8313
4. Use item.
5. Observe poison status.


## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
